babel/messages/jslexer.py

```diff
@@ -151,7 +151,7 @@ def unquote_string(string):
     return u''.join(result)
 
 
-def tokenize(source, jsx=True, dotted=True, template_string=True):
+def tokenize(source, jsx=True, dotted=True, template_string=True, lineno=1):
```

Comment:
Could you update the docstring for `lineno`?

Reply:
Done!